frontend/device/kobo/device.lua

```diff
@@ -310,6 +313,16 @@ function Kobo:init()
     }
     self.wakeup_mgr = WakeupMgr:new()
 
+    -- Tweak initial slot, if necessary
+    if self.main_finger_slot then
+        self.input.cur_slot = self.main_finger_slot
+        self.input.ev_slots = {
+            [self.main_finger_slot] = {
```

Comment:
does this needs the [] ??.

Reply:
c.f., https://github.com/koreader/koreader/blob/ac3c97af49e62c792cc7bdcfc51603c6289c63ed/frontend/device/input.lua#L144-L148

It's a *new* nested table, with hashes failing to mimic arrays ^^.